lib/graphql/queries.js

```diff
@@ -325,6 +325,8 @@ export const getCollectiveToEditQueryFields = `
     collectiveLimit
     addFundsLimit
     hostDashboard
+    hostedCollectivesCount
+    totalAddedFunds
```

Comment:
I don't see these properties in the matching API PR. Can you check?

Also, can we make it more consistent? `collectiveLimit` vs `hostedCollectivesCount` and `addFundsLimit` and `totalAddedFunds`